lib/app/controllers/inline_forms_controller.rb

```diff
@@ -271,4 +284,7 @@ def unravel(deep_hash, level=-1)
     a
   end
 
+  def revert_params
+    params.require(:id).permit(:update, :comchannel)
```

Comment:
Hmm it's weird to have comchannel here, don't remember why. It's not part of inline_forms (IF from now on) but from specifice implementations (i.e. business logic). Probably needed it to 'fix' something  :-1: